Cargo.toml

```diff
@@ -18,6 +18,8 @@ regex = "^1.0"
 relm = "0.24"
 relm-derive = "0.24"
 xdg = "2.1"
+dirs = "4.0.0"
+mktemp = "0.5.0"
```

Comment:
You should add dev dependencies in the [[dev-depedencies]](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies) section.